extensions/mssql/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.68",
+	"version": "v3.0.0-release.81",
```

Comment:
is the leading character 'v' intentionally added?

Reply:
absolutely not, thanks for the save! missed the fixing commit.